kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/model/ProxyModel.java

```diff
@@ -22,6 +21,6 @@
  */
 public record ProxyModel(ProxyResolutionResult resolutionResult,
```

Comment:
Is a `ProxyModel` really a "`ResolvedProxy`"?

Reply:
I'm using `Resolved` to mean 100% of the recursive dependencies could be resolved (putting aside the fact it doesn't consider Secrets/Configmaps yet). 

`ProxyModel` contains information about problems/unresolved referents in the ProxyResolutionResult.

Maybe like you're getting at later we should have everything operate on the `*ResolutionResult` and drop the `Resolved*` conveniences.

Reply:
have removed the `Resolved*` classes